frontend/packages/kubevirt-plugin/src/constants/vm/constants.ts

```diff
@@ -72,7 +72,7 @@ export const VIRTIO_WIN_IMAGE = 'virtio-win-image';
 
 export const PENDING_RESTART_LABEL = '(pending restart)';
 export const getPVCUploadURL = (pvcNamespace: string): string =>
-  `/k8s/ns/${pvcNamespace}/persistentvolumeclaims/~new/upload-form`;
+  `/k8s/ns/${pvcNamespace}/persistentvolumeclaims/~new/form`;
```

Comment:
why?
isn't `form` being used by the regular pvc form?

Reply:
im not sure I understand what you mean, but this change is about making them consistent with all ~new console requirements